apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -785,6 +808,10 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
                     formValuesToMutationDataPreprocessor={(values) => {
                         values.property = selectedProperties[0]
 
+                        if (get(values, 'hasAllProperties', null) && get(values, 'property', null)) {
```

Comment:
if you do not specify the third parameter in get, then it will return undefined, I think the behavior in this case will be the same as with returning null, but shorter